sinks/splunk/splunk.go

```diff
@@ -123,25 +179,26 @@ func NewSplunkSpanSink(server string, token string, localHostname string, valida
 	}
 
 	return &splunkSpanSink{
+		batchSize:          splunkConfig.HecBatchSize,
+		connLifetimeJitter: splunkConfig.HecConnectionLifetimeJitter,
 		hec:                client,
+		hostname:           config.Hostname,
 		httpClient:         httpC,
 		ingest:             make(chan *Event),
-		hostname:           localHostname,
-		log:                log,
-		sendTimeout:        sendTimeout,
-		ingestTimeout:      ingestTimeout,
-		batchSize:          batchSize,
-		spanSampleRate:     int64(spanSampleRate),
+		ingestTimeout:      splunkConfig.HecIngestTimeout,
+		log:                logger,
+		maxConnLifetime:    splunkConfig.HecMaxConnectionLifetime,
+		name:               "splunk",
```

Comment:
Should we be passing the `name` argument here?

Reply:
Good catch! Thanks, done!